File: Semestres/10.Agosto-Diciembre 2021/FADAUSB/25-Sep/Ejemplo2.py
import math

def recurrencia(n):
    if n==1:
        return 1
    else:
        return 8*recurrencia(n/4)+2*n**2

def formula(n):
    return 2*n**2+n**2*(0.5**((math.log2(n)/math.log2(4))-1)-1)/(-0.5)+n**(math.log2(8)/math.log2(4))


n = [4**i for i in range(0,10)]

for valor in n:
        print("n "+str(valor)+" RR "+str(recurrencia(valor))+ " Formula " + str(formula(valor)))

File: Semestres/9.Noviembre 2020-Abril 2021/MDII/22Enero2021/ejemplo2.rb

def formula(n)
  return (1271.0/64.0)-(631.0/64.0)*(5**n)-(1.0/8.0)*(n**2)-(7.0/16.0)*n+10*n*(5**n)
end

def recurrencia(n)
  if n == 0
    return 10
  elsif n == 1
    return 20
  else
    return 6*recurrencia(n-1)-5*recurrencia(n-2)+n+8*(5**n)
  end
end

20.times{
    |i|
    print("Valor n ",i," Recurrencia: ", recurrencia(i)," Formula: ", formula(i),"\n")
}
